R/utils_tidy_data.R

```diff
@@ -192,12 +198,22 @@ summarise_trips_without_trips <- function(participants, cons) {
       last_trip_local_datetime = format(max(lubridate::as_datetime(end_local_time)), usetz = FALSE)
     ), by = user_id] %>%
     .[, n_days := round(as.numeric(difftime(last_trip_datetime, first_trip_datetime, units = "days")), 1)]
-
+  
+  # Add the 'unconfirmed' column.
+  unconfirmed_summ <- trip_query %>%
+    .[is.na(trip_query[[confirmed_user_input_column]]), .(unconfirmed = .N), by = user_id]
+  
+  
+  # Count the number of trips per user
   n_trips <- count_total_trips(cons)
   summ_trips <- merge(n_trips, summ_trips, by = "user_id")
 
   message("merging trip summaries with participants")
-  merge(participants, summ_trips, by = "user_id", all.x = TRUE)
+  merge(participants, summ_trips, by = "user_id", all.x = TRUE) %>% 
+    merge(., unconfirmed_summ, by="user_id", all.x = TRUE) %>%
+    .[is.na(unconfirmed), unconfirmed := 0]
```

Comment:
Note to self: checked that this is the same as in my `master_specific_customization` branch